src/apps/content-editor/src/app/views/ItemEdit/components/Header/Header.less

```diff
@@ -3,7 +3,7 @@
 
 .Header {
   padding: 16px 32px;
-  z-index: 10;
+  z-index: 120;
```

Comment:
What is the logic behind the choice of the specific value of 120?

Reply:
@shrunyan dropdown is 110 so I went with 120 and made modalAligner 130 following zesty's concept of increments by 10 base.